index.d.ts

```diff
@@ -0,0 +1,45 @@
+import { TextStyle } from "react-native";
+import React from "react";
+
+/**
+ * Escape characters which are meaningful in regular expressions
+ */
+export type AutoEscape = boolean;
+
+/**
+ * Styles applied to highlighted text
+ */
+export interface HighlightStyle extends TextStyle {}
+
+/**
+ * Process each search word and text to highlight before comparing (eg remove accents);
+ */
+export type Sanitize = (text: string) => string;
+
+/**
+ * Array of search words
+ */
+export type SearchWords = string[];
+
+/**
+ * Styles applied to the text wrapper
+ */
+export interface Style extends TextStyle {}
```

Comment:
Couldn't that be `StyleProp<TextStyle>` to also support arrays?
Why are you using `interface` here instead of `type`?
And do they really need to be exported all?

Reply:
Man, definitely you're right. I really made an extra work that doesn't make any sense.